query/config_traversal.go

```diff
@@ -34,6 +31,27 @@ func TraverseConfig(ctx context.Context, id, relationType, direction string) []m
 	return configItems
 }
 
+// Fetch config IDs which match the type and direction upto max depth (5)
+func getRelatedTypeConfigID(ctx context.Context, ids []string, relatedType, direction string) []string {
+	var allIDs []string
+	for _, id := range ids {
+		q := ctx.DB().Table("related_configs_recursive(?, ?)", id, direction).Select("id", "depth", "type").Where("type = ?", relatedType)
```

Comment:
@moshloop Using related_configs_recursive function as its max depth is more and allows us to do skip level relation matching